tests/scaletest/scale_test.go

```diff
@@ -769,6 +777,113 @@ func UpdateIngressParallelWithNodeReboot(t *testing.T) {
 	REBOOTNODE = false
 }
 
+func CreateServiceTypeLBWithApp(t *testing.T, numPods int, numOfServices int, appNameLB string, serviceNamePrefixLB string, aviObjPrefix string) []string {
+	g := gomega.NewGomegaWithT(t)
+	t.Logf("Creating a %v deployment with %v replicas", appNameLB, numPods)
+	err := lib.CreateApp(appNameLB, namespace, numPods)
+	if err != nil {
+		t.Fatalf("ERROR : Could not create deployment for service type LB support as %v", err)
+	}
+
+	t.Logf("Creating %v services of type LB", numOfServices)
+	servicesCreated, port, err := lib.CreateLBService(serviceNamePrefixLB, appNameLB, namespace, numOfServices)
+	if err != nil {
+		t.Fatalf("ERROR : %v", err)
+	}
+	t.Logf("Verifying AVI object creation...")
+	g.Eventually(func() bool {
+		var VSList []string
+		var poolList []string
+		/* Verifying pool creation*/
+		for _, svc := range servicesCreated {
+			VSList = append(VSList, aviObjPrefix+svc)
+			poolList = append(poolList, aviObjPrefix+svc+"--"+port)
+		}
+		pools := lib.FetchPools(t, AviClients[0])
+		var aviPoolList []string
+		for _, pool := range pools {
+			aviPoolList = append(aviPoolList, *pool.Name)
+		}
+		diffNum := len(DiffOfLists(poolList, aviPoolList))
+		if diffNum != initialNumOfPools {
+			return false
+		}
+		/* Verification of servers on pools */
+		for _, pool := range pools {
+			if strings.HasPrefix(*pool.Name, aviObjPrefix+serviceNamePrefixLB) == true {
+				if len(pool.Servers) != numPods {
+					return false
+				}
+			}
+		}
+		/* Verifying VS creation */
+		VSes := lib.FetchVirtualServices(t, AviClients[0])
+		var ingressVSList []string
```

Comment:
This name has to be changed from ingressVSList for service type LB.

Reply:
Will change it